akka-stream/src/main/scala/akka/stream/impl/io/TcpStages.scala

```diff
@@ -272,12 +274,6 @@ private[stream] object TcpConnectionStage {
 
       override def onUpstreamFailure(ex: Throwable): Unit = {
         if (connection != null) {
-          if (interpreter.log.isDebugEnabled) {
```

Comment:
Hm I thought I brought it back in this PR, seems not. Will add back, as discussed.